docs/src/pages/components/Map/WebMap.mdx

```diff
@@ -15,23 +15,23 @@ import Readme from 'backpack/packages/bpk-component-map/README.md';
 ## Default
 
 <PresentationBlock>
-  <SimpleExample />
+  <iframe width="800" height="600" src="https://backpack.github.io/storybook/iframe.html?id=bpk-component-map--simple"></iframe>
 </PresentationBlock>
 
 ## Icon markers
 
 Use icon markers to display points of interest on a map. By default, they appear in Monteverde. When focused, they grow in size and turn white with a Monteverde border.
 
 <PresentationBlock>
-  <WithIconMarkersExample />
+  <iframe width="800" height="600" src="https://backpack.github.io/storybook/iframe.html?id=bpk-component-map--icon-markers"></iframe>
 </PresentationBlock>
 
 ## Price markers
 
 Use price markers to display a collection of prices on a map. By default, they appear in sky blue. When focused, they grow in size and turn white, and when viewed they turn light blue.
 
 <PresentationBlock>
-  <WithPriceMarkersExample />
+  <iframe width="800" height="600" src="https://backpack.github.io/storybook/iframe.html?id=bpk-component-map--price-markers"></iframe>
```

Comment:
So that it doesn't break out of its container when on mobile can you change the width to be:

```suggestion
  <iframe style={{width: '100%'}} height="600" src="https://backpack.github.io/storybook/iframe.html?id=bpk-component-map--simple"></iframe>
```